templates/base/src/pages/index.tsx

```diff
@@ -0,0 +1,207 @@
+import { NextPage } from 'next';
+import React from 'react';
+import { Button } from "ui";
+
+const Home: NextPage = () => {
+  return (
+    <div className="container">
+      <div>
+        <title>Create Next App</title>
+        <link rel="icon" href="/favicon.ico" />
+      </div>
+
+      <main>
```

Comment:
Too much content in this route lets stick to our told style of just returning a template from a route